### ISSUE TITLE
Environment variables are not replaced in resulting Info.plist

### ISSUE THREAD

**Issue by ondrejnovotny** [dansiegel/Mobile.BuildTools#287]
# Description
Environment variables are not replaced in Info.plist

## Reproduction Steps
 
1. Create new Xamarin.Forms app
2. Install Mobile.BuildTools (leave default buildtools.json)
3. Define

` "environment": {
    "defaults": {
      "AppId": "com.something.not.working" }
  }
},`

4. Set to Debug, Simulator
5. Build

Expected 

## Expected Behavior

Variable $AppId$ is replaced with "com.something.not.working" in "\obj\iPhoneSimulator\Debug\mbt" folder

## Actual Behavior

Variable is not replaced

## Environment

- OS: [Windows 10|
- IDE: [VS2019|
- BuildTools Version: [2.x.x]
- IDE Version: [17.x.x]
- Build Target Framework: [netstandard2.0|Xamarin.iOS|

**Comment by themronion** [comment]
@ondrejnovotny have u managed to find a solution?

**Comment by ondrejnovotny** [comment]
> @ondrejnovotny have u managed to find a solution?

No, I have ended with replacing all manifests in pre-build events.
